go.html.markdown

```diff
@@ -472,4 +473,9 @@ documentation](http://golang.org/pkg/) and the source code comes up!
 
 Another great resource to learn Go is [Go by example](https://gobyexample.com/).
 
+There are many excellent conference talks and video tutorials on Go available on YouTube, and Go trainer [John Arundel](https://bitfieldconsulting.com/golang) has put together three playlists of the very best, tailored for beginners, intermediate, and advanced Gophers respectively:
```

Comment:
The video playlists are fine for me, but I think the link to the commercial training of the author need to be removed. What do you think @divayprakash ?

Reply:
I agree completely. Also, the author of the PR has put in the same link in the YAML header also, which would need to be removed too.

Reply:
yep, makes sense.

@bitfield

Reply:
> the author of the PR has put in the same link in the YAML header also

Sorry about that, I saw that previous contributors had included web links too, and assumed that it was OK for me to do that. Is it not?

Reply:
The link to your Github account is okay, the link to your commercial training offers is not. Learn X in Y minutes is not there to create traffic to your website.

As long as the sentence "and Go trainer John Arundel of Bitfield Consulting" and the link "https://bitfieldconsulting.com/golang" is in the PR, I will not merge this.

People who click on the playlist link will see the name of your company, so will people who look at your Github profile.

Reply:
That's perfectly fine! I wasn't arguing with you, just asking for information. As you'll appreciate, it's not always completely clear for new contributors to a project what the rules are, especially if they're not actually written down anywhere. Thanks for the clarification, and I've made the changes you asked for.

Reply:
Thanks for making the changes, I'll merge this now.

Reply:
Much appreciated, thank you!